.github/workflows/deploy.yaml

```diff
@@ -88,10 +88,47 @@ jobs:
         env:
           HOPR_PACKAGE: hoprd
 
+  deploy_smart_contracts:
+    name: Deploy Smart Contracts
+    runs-on: ubuntu-latest
+    needs: [build]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure Git info
+        run: bash -x scripts/configure-git-info.sh
+
+      - name: Update repo from latest pushes.
```

Comment:
This should not be done in a workflow since it moves the target of the whole workflow and makes it non-deterministic.